backend/src/nodes/utils/pytorch_auto_split.py

```diff
@@ -53,7 +53,6 @@ def auto_split_process(
         d_img = None
         try:
             device = torch.device(os.environ["device"])
-            model = model.to(device)
```

Comment:
Why did you remove this?

Reply:
Because it's already on the GPU when it loads the model in. this is unnecessary. At best it's a no-op, at worst it just wastes time. It never moves it off the device, so doing it once is all it needs.